examples/howto/plot_connectivity.py

```diff
@@ -115,16 +120,16 @@ def get_network(probability=1.0):
 
 ###############################################################################
 # We can plot the sparse connectivity pattern between cell populations.
-conn_idx = len(net_sparse.connectivity) - 1
-plot_connectivity_matrix(net_sparse, conn_idx)
-
-conn_idx = len(net_sparse.connectivity) - 2
+src_gids, target_gids = 'L2_basket', 'L2_basket'
+loc, receptor = 'soma', 'gabaa'
+conn_indices = net_erp.pick_connection(src_gids, target_gids, loc, receptor)
```

Comment:
```suggestion
conn_indices = net_erp.pick_connection(src_gids='L2_basket', target_gids='L2_basket, loc='soma', receptor='gabaa')
```
I think this is more pythonic as it uses keyword arguments which are less error-prone